Documentation/cluster-tpr.md

```diff
@@ -1,35 +1,48 @@
+  - `deviceFilter`: A regular expression that allows selection of devices to be consumed by OSDs.  If individual devices have been specified for a node then this filter will be ignored.  This field uses [golang regular expression syntax](https://golang.org/pkg/regexp/syntax/). For example:
+    - `sdb`: Only selects the `sdb` device if found
+    - `^sd.`: Selects all devices starting with `sd`
+    - `^sd[a-d]`: Selects devices starting with `sda`, `sdb`, `sdc`, and `sdd` if found
+    - `^s`: Selects all devices that start with `s`
+    - `^[^r]`: Selects all devices that do *not* start with `r`
+  - `metadataDevice`: Name of a device to use for the metadata of OSDs on each node.  Performance can be improved by using a low latency device (such as SSD or NVMe) as the metadata device, while other spinning platter (HDD) devices on a node are used to store data.
+  - `location`: Location information about the cluster to help with data placement, such as region or data center.  This is directly fed into the underlying Ceph CRUSH map.  More information on CRUSH maps can be found in the [ceph docs](http://docs.ceph.com/docs/master/rados/operations/crush-map/).
+  - `storeConfig`: Configuration information about the store format for each OSD.
+    - `storeType`: `filestore` or `bluestore` (default: `filestore`), The underlying storage format to use for each OSD.
+    - `databaseSizeMB`:  The size in MB of a bluestore database.
+    - `walSizeMB`:  The size in MB of a bluestore write ahead log (WAL).
+    - `journalSizeMB`:  The size in MB of a filestore journal.
+  - `nodes`: Names of individual nodes in the cluster that should have their storage included in accordance with either the cluster level configuration specified above or any node specific overrides described in the next section below.
 
 ### Node settings
-Coming soon we will have more granular settings for nodes, devices, topology, etc.
\ No newline at end of file
+In addition to the cluster level settings specified above, each individual node can also specify configuration to override the cluster level settings and defaults.  If a node does not specify any configuration then it will inherit the cluster level settings.  In the list below, if a description for a field is blank then please refer to its description in the cluster level settings section above.
+- `name`: The name of the node, which should match its `kubernetes.io/hostname` label.
+- `devices`: A list of individual device names belonging to this node to include in the storage cluster.
+  - `name`: The name of the device (e.g., `sda`).
+- `directories`:  A list of directory paths on this node that will be included in the storage cluster.  Note that using two directories on the same physical device can cause a negative performance impact.
+  - `path`: The path on disk of the directory (e.g., `/rook/storage-dir`).
+- `useAllDevices`
```

Comment:
I played around with moving the duplicated settings to their own sections and linking to them, not sure how I feel about it though.  What do you think of this flow? https://github.com/jbw976/rook/blob/storage-config-docs/Documentation/cluster-tpr.md#cluster-settings